canonical/ubuntu-pro-client
# i18n prep: logging: standardize logging callsite conventions (SC-1602)

## Why is this needed?
This commit standardizes our logging calls in a couple ways:
 - log messages should be inline strings at the callsite
 - log string formatting should use lazy % style
 - logs shouldn't share strings with printed messages

In the places where log and print calls shared a string, I separated
them and attempted to make the log string more "log-like" and the
printed string more user-friendly. I also moved the printed string
to the messages module.

## Test Steps
Please review every log call and make sure it follows these conventions now.
Please review the changes in messages.

CI should pass

## Checklist

 - [n/a] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?
